infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationDao.kt

```diff
@@ -192,6 +207,8 @@ class PublicationDao(
                 on switch_joint_version.switch_id = coalesce(draft_switch.draft_id, draft_switch.official_id)
                   and switch_joint_version.switch_version = coalesce(draft_switch.draft_version, draft_switch.official_id)
                   and switch_joint_version.number = switch_structure.presentation_joint_number
+             left join splits 
```

Comment:
Näitähän pitäisi tietenkin tulla 0 tai 1, mutta taulurakenne mahdollistaisi periaatteessa useammankin. Pitäisiköhän tuonne resultsetin käsittelyyn laittaa sanity-checkinä require että sama publishcandidate (id) esiintyy vain kerran. Saa kiinni jos jossain joineissa tulee vahingossa monistettua rivit.